Library/brew.sh

```diff
@@ -201,6 +169,64 @@ EOS
   esac
 fi
 
+if [[ -z "$HOMEBREW_DEVELOPER" ]]
+then
+  unset HOMEBREW_RUBY_PATH
+fi
+
+if [[ -z "$HOMEBREW_RUBY_PATH" && "$HOMEBREW_COMMAND" != "install-vendor" ]]
```

Comment:
Sorry, missed your note. I think it would be good to put this in a function and only use it for Ruby commands.